cmd/ocm/create/cluster/cmd.go

```diff
@@ -1036,8 +1137,15 @@ func promptExistingVPC(fs *pflag.FlagSet, connection *sdk.Connection) error {
 	return err
 }
 
-func promptCCS(fs *pflag.FlagSet) error {
-	err := arguments.PromptBool(fs, "ccs")
+func promptCCS(fs *pflag.FlagSet, isGcpSubscriptionType bool) error {
+	var err error
+	// If marketplace-gcp subscription type is used, ccs should by default be true
+	if isGcpSubscriptionType {
+		fmt.Println("setting ccs to 'true'")
+		args.ccs.Enabled = true
```

Comment:
We could move this logic outside this function

Reply:
Moved outside this function and passed a value on whether to prompt for ccs or not